src/model/documentfragment.js

```diff
@@ -235,6 +235,11 @@ export default class DocumentFragment {
 		nodes = normalize( nodes );
 
 		for ( const node of nodes ) {
+			// If node that is being added to this element is already inside another element, first remove it from the old parent.
+			if ( node.parent !== null ) {
```

Comment:
I'm hesitating whether I'd rather like `if ( node.parent )` or `!== null` ;| Dunno

Reply:
`!== null` is easier to find when searching codebase